back/spec/acceptance/invites_spec.rb

```diff
@@ -282,7 +282,6 @@
         boulettos = json_response.dig(:included).select{|inc| inc[:id] == invite.invitee.id}&.first
         expect(boulettos&.dig(:attributes,:last_name)).to eq('Boulettos')
         expect(boulettos&.dig(:attributes,:invite_status)).to eq('accepted')
-        expect(invite.reload.invitee.registration_completed_at).to be_present  # when no custom fields
```

Comment:
This is fine, but slightly better would be to change the test to make sure it's _not_ set instead.

Reply:
I have changed this to use `... .to eq(nil)` which seems clearer than using `... .not_to be_present`